api/main.py

```
@@ -22,6 +23,12 @@ def handler(event, context):
         response = asgi_handler(event, context)
         return response
 
+    elif "Records" in event:
+        for record in event.get("Records", []):
+            if "s3" in record:
+                storage.get_object(record)
+        return "Success"
```

If we receive an SNS event, I think it also has Records but wouldn't have the s3 key, this would still return Success in that case when we didn't process any events.

---

You are right - better to be explicit than fail silently

---

It now logs that it did not know how to handle an event

---

nit: s/recieved/received